README.md

```diff
@@ -0,0 +1,6 @@
+1. Copy your msso_config.json file inside platforms/android/assets folder
+2. Copy the MAS libraries inside platforms/android/libs folder. Name the libraries as mentioned in the build.gradle file.
+
+```
+Disclaimer: This app is build using Cordova version 7.1.0. It won't compile if using Cordova Version 8.0, since Cordova Android has changed the internal structure of Android folders and this app doesn't conforms to that design. 
```

Comment:
This app is built using Cordova version 7.1.0. The app will not compile with Cordova version 8.0, because Cordova Android changed the internal structure of Android folders.